### PR TITLE
Marketplace request APIs

### PR DESCRIPTION
**Pull request by Ajay-Maury**
https://docs.google.com/document/d/1NPVE1DnN1k3TL08vAeJ88dCZRUAjii4j577iGShFn28/edit?usp=sharing

### CODE REVIEW

**Review by kh4l1d64 — CHANGES_REQUESTED** [review]
Need Unit tests but other than that LGTM

File: src/request/dto/create-request.dto.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import { RequestStatusEnum, RequestTypeEnum } from '@prisma/client';
```
Comment (VamshiBatta07) [inline src/request/dto/create-request.dto.ts:1]: Need to import the ENUMs from a different folder.
path: "src/request/enum/request.enum.ts"

Comment (Ajay-Maury) [reply src/request/dto/create-request.dto.ts:1]: tried that also but that was creating datatype issue for Prisma queries

**Comment by VamshiBatta07** [comment]
To be added: 

- [ ] Unit test cases
- [x] Validation Pipes
- [x] Docker setup. 

Otherwise it looks good.

File: src/request/entities/request.entity.ts
Comment (VamshiBatta07) [inline src/request/entities/request.entity.ts:1]: We are not using the Request entity anywhere. Is there a future future use case ?

Comment (Ajay-Maury) [reply src/request/entities/request.entity.ts:1]: no use for now if we are not shifting to SQL to NOSQL DB in future

**Review by SanchitEsMagico — APPROVED** [review]

**Review by kh4l1d64 — APPROVED** [review]

**Pull request merged**